maven-resolver-provider/src/test/java/org/apache/maven/repository/internal/DefaultArtifactDescriptorReaderTest.java

```diff
@@ -27,25 +27,39 @@
 import org.eclipse.aether.artifact.DefaultArtifact;
 import org.eclipse.aether.impl.ArtifactDescriptorReader;
 import org.eclipse.aether.impl.RepositoryEventDispatcher;
+import org.eclipse.aether.repository.RemoteRepository;
 import org.eclipse.aether.resolution.ArtifactDescriptorRequest;
 import org.mockito.ArgumentCaptor;
 
 public class DefaultArtifactDescriptorReaderTest
     extends AbstractRepositoryTestCase
 {
 
-    public void testMng5459()
+    DefaultArtifactDescriptorReader reader;
```

Comment:
These can be private

Reply:
> 
> 
> These can be private

I agree, make this and the subsequent ones private

Reply:
Marked them as private.